lookout/style/format/cmdline_tools.py

```diff
@@ -73,7 +73,7 @@ def add_rules_thresholds(my_parser: ArgumentParser):
 
     :param my_parser: Parser to add the arguments to.
     """
-    my_parser.add_argument("--confidence-threshold", type=float, default=0.95,
+    my_parser.add_argument("--conf-threshold", type=float, default=0.95,
```

Comment:
"conf" can stand for "config", too, so better to leave the old name.

Reply:
we used conf in the Rule NamedTuple though

Reply:
Yes, given the Rule context that's fine. Here we've got command line arguments.